### DISCUSSION TITLE
Fleet Node in DMZ

### DISCUSSION THREAD

**Discussion by rahatUlAin** [Security-Onion-Solutions/securityonion#12379] [Unsupported Versions]
Can Fleet node retain data for the time of connection loss with the manager? I am using fleet node with standalone manager and sometimes the two systems stop communicating.  The firewall shows forcedly closed connections. I end up restarting both to establish communication.

**Comment by reyesj2** [comment]
You might benefit from having a receiver node setup for the event of manager connection going down https://docs.securityonion.net/en/2.4/architecture.html#receiver-node
